flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/JoinToMultiJoinForReorderRule.java

```diff
@@ -99,24 +99,24 @@
  * @see CoreRules#JOIN_TO_MULTI_JOIN
  */
 @Value.Enclosing
-public class FlinkJoinToMultiJoinRule extends RelRule<FlinkJoinToMultiJoinRule.Config>
+public class JoinToMultiJoinForReorderRule extends RelRule<JoinToMultiJoinForReorderRule.Config>
```

Comment:
update JavaDoc?
```
This rule is copied from {@link org.apache.calcite.rel.rules.JoinToMultiJoinRule}
```

Reply:
This is the rule used for reordering and I just tried to give it a better name. The javadoc is still accurate since I change it it. The rule is mostly copied from {@link org.apache.calcite.rel.rules.JoinToMultiJoinRule} with some modifications.